setup/ConfigureSwift.rb

```diff
@@ -18,7 +18,7 @@ def perform
       case framework
         when :quick
           configurator.add_pod_to_podfile "Quick', '~> 1.2.0"
-          configurator.add_pod_to_podfile "Nimble', '~> 7.0.2"
+          configurator.add_pod_to_podfile "Nimble', '~> 7.3.1"
```

Comment:
can also do `~> 7.0`

Reply:
Confirming that 7.0 also fixes the build issues.

Reply:
@brianLeeson I know this is an old PR but can you update this so we can merge it?

Reply:
Forgive me, but I don't know how to do that.  Do you mean mark as resolved? Pull the most recent code changes? It looks like there are no merge conflicts. I'm happy to take the steps, I just don't know what they are...

Reply:
Can you change this to do `~> 7.0` and push to this branch?